scripts/deploy.backend.sh

```diff
@@ -14,18 +14,67 @@ case $ENV in
     ;;
 esac
 
+II_CANISTER_ID="$(dfx canister id internet_identity --network "$ENV")"
```

Comment:
This is not incorrect for a deployment on mainnet (production and staging)?

Reply:
Why would this be incorrect for production and staging?

Reply:
Right, it works for production. Doesn't for staging though

> ❯ dfx canister id internet_identity --network staging
> Error: Cannot find canister id. Please issue 'dfx canister create internet_identity --network staging'.